pom.xml

```diff
@@ -77,6 +77,7 @@
     <libcal.auth.retry.count></libcal.auth.retry.count>
     <libcal.auth.retry.delay></libcal.auth.retry.delay>
     <libcal.auth.expires_in.padding></libcal.auth.expires_in.padding>
+    <allowed.ips></allowed.ips>
```

Comment:
Would probably be good to add the `libcal.` prefix (likewise `LIBCAL_` for env vars) since that seems to be a convention we're using.

Reply:
Added `libcal` prefix.